idl/storage_proxy.idl.hh

```diff
@@ -21,8 +21,9 @@
 #include "idl/per_partition_rate_limit_info.idl.hh"
 #include "idl/keys.idl.hh"
 #include "idl/uuid.idl.hh"
+#include "idl/storage_service.idl.hh"
 
-verb [[with_client_info, with_timeout, one_way]] mutation (frozen_mutation fm, inet_address_vector_replica_set forward, gms::inet_address reply_to, unsigned shard, uint64_t response_id, std::optional<tracing::trace_info> trace_info [[version 1.3.0]], db::per_partition_rate_limit::info rate_limit_info [[version 5.1.0]]);
+verb [[with_client_info, with_timeout, one_way]] mutation (frozen_mutation fm, inet_address_vector_replica_set forward, gms::inet_address reply_to, unsigned shard, uint64_t response_id, std::optional<tracing::trace_info> trace_info [[version 1.3.0]], db::per_partition_rate_limit::info rate_limit_info [[version 5.1.0]], service::fencing_token fence [[version 5.3.0]]);
```

Comment:
Won't make it into 5.3.

Reply:
It tells idl compiler that the parameter should be rpc::optional.

Reply:
It is a documentation thing. Yes.

Reply:
@bhalevy also plans to use them to generate IDL signatures so that we can mechanically check compatibility.